packages/rpc/src/adaptors/iframe-child.ts

```diff
@@ -7,12 +7,31 @@ export function fromInsideIframe({targetOrigin = '*'} = {}): MessageEndpoint {
     );
   }
 
+  /**
```

Comment:
Nit: these aren't documentation, so we generally use single-line comments for them instead.